### PR TITLE
chore: enforce files naming to use .styles.ts [teams-prg files]

### PR DESCRIPTION
**Pull request by layershifter**
## New Behavior

This PR removes `useCOMPONENTStyles.ts` files to `useCOMPONENTStyles.styles.ts` files to follow [Griffel's rule for naming files](https://github.com/microsoft/griffel/blob/main/packages/eslint-plugin/src/rules/styles-file.md).

The ESLint rule have not been enabled yet, this PR is a preparation to do that. Affects files owned by @microsoft/teams-prg 

Change files are set to patch/prerelease as artifacts should be updated.

## Related Issue(s)

- Related to #27672

### CODE REVIEW

File: packages/react-components/react-menu/src/components/Menu/Menu.test.tsx [hunk 1]
```diff
@@ -144,7 +144,6 @@ describe('Menu', () => {
         </MenuTrigger>
         <MenuPopover>
           <MenuList>
-            {/* eslint-disable-next-line @fluentui/max-len */}
```
Comment (layershifter) [inline packages/react-components/react-menu/src/components/Menu/Menu.test.tsx:147]: Oops, not needed

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/Avatar-mount_base.html">707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/Avatar-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/Button-mount_base.html">389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/Button-mount.html">393</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/Field-mount_base.html">1220</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/Field-mount.html">1237</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/FluentProvider-mount_base.html">781</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/FluentProvider-mount.html">783</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/FluentProviderWithTheme-mount_base.html">103</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/FluentProviderWithTheme-mount.html">110</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">99</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">89</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">104</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">98</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/InfoButton-mount_base.html">20</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/InfoButton-mount.html">17</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/MakeStyles-mount_base.html">1184</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/MakeStyles-mount.html">1180</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/Persona-mount_base.html">1926</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/Persona-mount.html">1880</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/SpinButton-mount_base.html">1483</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27698/perf-test-react-components/SpinButton-mount.html">1460</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                 Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | -----------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `204.498 kB`<br />`57.184 kB` | `204.498 kB`<br />`57.182 kB` | <br />`-2 B`  |
| <samp>react-datepicker-compat</samp> <br /> <abbr title='bundle-size/DatePicker.fixture.js'>DatePicker Compat</abbr>                                                          | `220.414 kB`<br />`58.527 kB` | `220.414 kB`<br />`58.538 kB` | <br />`11 B`  |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `128.088 kB`<br />`39.045 kB` | `128.088 kB`<br />`39.044 kB` | <br />`-1 B`  |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `128.359 kB`<br />`39.232 kB` | `128.359 kB`<br />`39.231 kB` | <br />`-1 B`  |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |   `11.649 kB`<br />`4.263 kB` |   `11.649 kB`<br />`4.262 kB` | <br />`-1 B`  |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `46.819 kB`<br />`16.412 kB` |  `46.819 kB`<br />`16.413 kB` |  <br />`1 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                |          Size (minified/GZIP) |
| ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                          |  `93.779 kB`<br />`22.493 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                       |  `57.544 kB`<br />`14.966 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                             |   `15.632 kB`<br />`6.258 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                     |  `73.758 kB`<br />`19.478 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                    |  `87.056 kB`<br />`28.011 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                    |   `85.74 kB`<br />`27.747 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> |   `65.31 kB`<br />`17.931 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>               |    `36.086 kB`<br />`11.9 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                       |  `91.028 kB`<br />`27.083 kB` |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoLabel.fixture.js'>InfoLabel</abbr>                                                             | `131.371 kB`<br />`40.039 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                | `131.343 kB`<br />`39.747 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                  |   `11.214 kB`<br />`4.271 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                    |  `64.465 kB`<br />`16.892 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                    | `115.071 kB`<br />`35.448 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                            |    `6.446 kB`<br />`2.186 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                     |   `18.033 kB`<br />`6.666 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                    | `147.821 kB`<br />`40.593 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                               |  `44.348 kB`<br />`12.347 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                    | `130.378 kB`<br />`33.193 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                            |  `78.289 kB`<br />`19.194 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                      |  `77.619 kB`<br />`19.004 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/e0e8de0dadce1a0adcd77acb96028d562773b53d'>e0e8de0dadce1a0adcd77acb96028d562773b53d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Review by bsunderhus — APPROVED** [review]

**Pull request merged**